frontend/packages/kubevirt-plugin/integration-tests/tests/models/editFlavorView.ts

```diff
@@ -0,0 +1,11 @@
+import { $ } from 'protractor';
+
+export const flavorDropdownId = '#vm-flavor-modal-flavor-dropdown';
+
+export const modalTitle = $('.modal-title');
```

Comment:
As per ES module spec, this file gets evaluated only once, when imported (or required).

I'd suggest moving `$(foo)` statements (which I assume depend on the _current_ DOM context) into functions.

```ts
export const modalTitle = () => $('.modal-title');
```

Reply:
done